.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Release Package
+on:
+  release:
+    types: [ released ]
+jobs:
+  publish-to-npm:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [14.6.0]
```

Comment:
Did you mean for these node versions to be different?

Reply:
Nope. Good catch.